docs/release-notes/unreleased/harmony-fallback-242.md

```
@@ -0,0 +1 @@
+- Align the build-time `Lib.Harmony` fallback package with the documented Harmony 2.4.2 native Apple Silicon workaround.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)